editor/components/page-attributes/order.js

```diff
@@ -39,13 +39,23 @@ export function PageAttributesOrder( { onUpdateOrder, instanceId, order } ) {
 				id={ inputId }
 				size={ 6 }
 			/>
-		</PageAttributesCheck>
+		</Fragment>
+	);
+}
+
+// TODO: Refactor to use a future ifPostTypeSupports HoC
```

Comment:
Trying out a HoC (per https://github.com/WordPress/gutenberg/pull/4662/#discussion_r163811780) sounds fine, but until then I don't think there's a point to keeping this comment.